src/extension/webviews/query_page/App.tsx

```diff
@@ -21,6 +21,7 @@
  * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
+import * as vscode from 'vscode';
```

Comment:
Using `vscode` like this doesn't work here, webviews don't have access to the `vscode` global, instead we're using a local `vscode` obtained via `useQueryVSCodeContext();`

Reply:
Thanks for pointing this out. I saw that usage but missed removing it here.